code/modules/mining/equipment/survival_pod.dm

```diff
@@ -145,20 +145,6 @@
 	if(!state_open)
 		. += "sleeper_cover"
 
-//Lifeform Stasis Unit
-/obj/machinery/stasis/survival_pod
```

Comment:
Will you replace this with a life support equivalent? This was a stasis bed with the original survival pod sleeper sprite.